doc/api/crypto.md

```diff
@@ -2057,6 +2081,68 @@ Based on the recommendations of [NIST SP 800-131A][]:
 
 See the reference for other recommendations and details.
 
+### CCM mode
+
+CCM is one of the two supported [AEAD algorithms][]. Applications which use this
+mode must adhere to certain restrictions when using the cipher API:
+
+- The authentication tag length must be specified during cipher creation by
+  setting the `authTagLength` option.
+- The length of the initialization vector (nonce) must be one of 4, 6, 8, 10,
+  12, 14, or 16 bytes.
+- When decrypting, the authentication tag must be set via `setAuthTag()` before
+  specifying additional authenticated data and / or calling `update()`.
+  Otherwise, decryption will fail and `final()` will throw an error. Note that
+  decryption without setting the correct authentication tag is currently
+  unsupported in compliance with section 2.6 of [RFC 3610][].
```

Comment:
This sentence is not needed because 2.6 of RFC3610 only says size limit of CBC-MAC and mandating verification of CBC-MAC before decryption.
`currently unsupported` would suggest users that CCM without verification might be supported in the future.

Reply:
Other crypto libraries allow to decrypt messages even if the authentication tag is missing or incorrect and leave this to the user, that's why I put "currently unsupported" here.